content/en/tracing/trace_search_and_analytics/_index.md

```diff
@@ -19,9 +19,11 @@ The Datadog app shows a **Live** indicator next to the time selector whenever yo
 
 {{< img src="tracing/live_search/LiveSearch.png" alt="Live Search Indicator" >}}
 
-All ingested traces are passed through custom [retention filters][3] that you can create to determine which spans to index, along with the default [intelligent retention filter][4] that retains a diverse set of traces.
+All ingested traces are passed through [custom retention filters][3] that you can create to determine which spans to index, along with the default [intelligent retention filter][4] that retains a diverse set of traces.
 
-Once indexed, traces are available for use in Search and Analytics, and they are retained for 15 days.
+Once indexed through a custom retention filter, traces are available for use in Search and Analytics, and they are retained for 15 days.
+
+When indexed through the intelligent retention filter, traces are available for use in Search (not Analytics, unless a custom filter has also retained them), and they are retained for 30 days.
```

Comment:
A span cannot be retained both by the intelligent retention filter and a given custom retention filter - If a span is matched by both, only the custom retention filter will index it